modules/bullet/collision_object_bullet.h

```diff
@@ -127,15 +128,18 @@ class CollisionObjectBullet : public RIDBullet {
 	VSet<RID> exceptions;
 
 	bool need_body_reload = true;
+	bool need_collision_fm_reload = true;
 
 	/// This array is used to know all areas where this Object is overlapped in
 	/// New area is added when overlap with new area (AreaBullet::addOverlap), then is removed when it exit (CollisionObjectBullet::onExitArea)
 	/// This array is used mainly to know which area hold the pointer of this object
-	Vector<AreaBullet *> areasOverlapped;
+	LocalVector<AreaBullet *> areasOverlapped;
 	bool isTransformChanged = false;
 
 public:
 	bool is_in_world = false;
+	bool into_flush = false;
+	bool into_process = false;
 
```

Comment:
`in_process` doesn't appear to be used. It can be removed.

`into_flush` appears to be used for marking whether the `CollisionObject` has been added to the either the queue to be flushed or the pre-flush queue. Is this intended, or should there be separate markers for each queue?

Finally, nitpicking, I think `into_flush` would be better named `is_in_queue`. And if there should be two separate markers then `is_in_pre_flush_queue` and `is_in_flush_queue`.

Reply:
Woups, `into_process` should not be there, and `into_flush` got renamed to `is_in_flush_queue`. Yes, is intended that we have only `is_in_flush_queue` because a `CollisionObject` can be an `Area` or a `RigidBody` - so can be in one or the other queue but never in both.